hlto2catracking.sh

```diff
@@ -0,0 +1,36 @@
+# Package for the HLT TPC O2 CA Tracking library in standalone build (This package is to be used if AliRoot is not in the dependency)
+package: HLTO2CATracking
+version: "%(tag_basename)s"
+tag: hlt_o2_ca_tracking-v1.0
+source: https://github.com/davidrohr/AliRoot
+build_requires:
+  - CMake
+  - GCC-Toolchain:(?!osx)
```

Comment:
maybe this even needs to be a real requirement (as opposed to build_requires) since the C++ libraries might be needed at runtime?